scripts/generic-ci-tests.sh

```diff
@@ -108,7 +108,10 @@ case "$TEST_SUITE" in
                 paver test_system -s lms --extra_args="--attr='shard_2' --with-flaky" --cov_args="-p"
                 ;;
             "3")
-                paver test_system -s lms --extra_args="--attr='shard_1=False,shard_2=False' --with-flaky" --cov_args="-p"
+                paver test_system -s lms --extra_args="--attr='shard_3' --with-flaky" --cov_args="-p"
```

Comment:
I just banged it out real quick in https://github.com/edx/edx-platform/pull/12286 (`sed` is a wonderful thing)